tpl/page_optm/settings_media.tpl.php

```diff
@@ -150,7 +150,7 @@
 		<td>
 			<?php $this->build_switch( $id ); ?>
 			<div class="litespeed-desc">
-				<?php echo sprintf( __( 'Automatically generate %s in the background via a cron-based queue.', 'litespeed-cache' ), __( 'Responsive Placeholder', 'litespeed-cache' ) ); ?>
+				<?php echo sprintf( __( 'Automatically generate %s in the background via a cron-based queue.', 'litespeed-cache' ), __( 'LQIP', 'litespeed-cache' ) ); ?>
```

Comment:
As LQIP is a special word, prolly no translation, no need sprintf anymore

Reply:
Good point, yeah